examples/PublishSubscribe/test_csharp_only.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+cd /Users/patdhlk/src/patdhlk/iceoryx2/iceoryx2-ffi/csharp/examples/PublishSubscribe
```

Comment:
Should not be an absolute path, maybe something like this:
```suggestion
cd "$(dirname "$(readlink -f "$0")")" || exit
```

Same in the other script